source/mobile/mobile-overview.rst

```diff
@@ -1,37 +1,85 @@
+|                                                                |                                                                     |  
+| - Can’t control the look, feel, and capabilities               |                                                                     |  
+|   of Mattermost Mobile Apps.                                   |                                                                     |
+| - Can’t deploy your own push proxy server.                     |                                                                     |  
++----------------------------------------------------------------+---------------------------------------------------------------------+
 
-.. note::
-    App wrapping via an EMM provider is untested, and not supported by Mattermost. Instead, we recommend using an EMM provider that supports Managed App Configuration.  
+Technical and Security Requirements
+-----------------------------------
+
+See our `Supported Devices/Mobile Device Requirements <https://docs.mattermost.com/install/requirements.html#mobile-apps>`__ documentation for basic mobile device requirements, and our `Supported Mattermost Server Versions <https://docs.mattermost.com/administration/mobile-changelog.html>`__ documentation for details on Mattermost Server minimum requirements. 
+
+We recommend running the latest version of the Mattermost Server as it contains the most recent features and applicable security updates. If this is not possible, we encourage you to be on the most recent Extended Support Release version of Mattermost. This release has critical feature updates that will ensure compatibility in a number of areas, including the Mattermost Push Notification Service (MPNS).
+
+Important
+Not all provided updates are compatible with all previous versions of Mattermost. Updating only Mattermost Mobile apps or updating the mobile apps before updating Mattermost Server can result in incompatibility issues.
+
+You should also start thinking about technical and security requirements in parallel with an implementation plan. Feel free to use our `implementation plan template <https://docs.mattermost.com/getting-started/implementation_plan.html>`__ available in the Mattermost documentation. 
+
+Also, start engaging your technical and security teams with the following questions:
+
+- Are there any known security or access requirements?
+- Is an `Enterprise Mobile Management (EMM) Provider <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html>`__ needed?
+- Is a `VPN connection required <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html#consider-mobile-vpn-options>`__?
+
+Asking questions like these requires you to pause before jumping into your Mattermost mobile project. However, it will help ensure you see a return on your investment.
+
+Getting Help
+------------
+
+If you need assistance, please reach out using one or more of the following methods.
 
```

Comment:
Suggestion - use colons for the list below (instead of hyphens), for consistency.

Reply:
Thanks! Great suggestion!